### PR TITLE
Add changes to rubocop settings. Fix documentation misspelling.

### PR DESCRIPTION
**Pull request by SBlazhko**
Add changes to rubocop settings. Fix documentation misspelling.

### CODE REVIEW

File: guides/code-style/.rubocop.yml [hunk 1]
```diff
@@ -1,10 +1,16 @@
-Metrics/LineLength:
-  Max: 80
```
Comment (IlyaUmanets) [inline guides/code-style/.rubocop.yml:2]: We need this configuration

File: guides/code-style/.rubocop.yml [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
+    - 'bin/*'
+    - 'vendor/*'
+    - './Rakefile'
+
+Metrics/LineLength:
+  Max: 80
 
 Metrics/MethodLength:
-  Enabled: false
+  Max: 15
```
Comment (IlyaUmanets) [inline guides/code-style/.rubocop.yml]: `Max: 10` is enough

**Review by IlyaUmanets — CHANGES_REQUESTED** [review]

File: guides/code-style/.rubocop.yml (hunk 1 above)
Comment (vitaliiprodan) [reply guides/code-style/.rubocop.yml:2]: We have it at line 10

File: guides/code-style/.rubocop.yml [hunk 3]
```diff
@@ -48,10 +44,6 @@ Style/IndentArray:
 Style/MultilineMethodCallIndentation:
   EnforcedStyle: indented
 
-Style/NumericLiterals:
```
Comment (vitaliiprodan) [inline guides/code-style/.rubocop.yml:51]: Why did you remove this?

File: guides/code-style/.rubocop.yml [hunk 4]
```diff
@@ -65,6 +57,13 @@ Style/PercentLiteralDelimiters:
     '%x': ()
 
 Style/StringLiterals:
-  Exclude:
```
Comment (vitaliiprodan) [inline guides/code-style/.rubocop.yml:68]: same

**Review by vitaliiprodan — COMMENTED** [review]
please use dashes instead of underscore for branch naming `add_some_changes_to_rubocop_settings` -> `add-some-changes-to-rubocop-settings`

File: guides/code-style/.rubocop.yml (hunk 1 above)
Comment (IlyaUmanets) [reply guides/code-style/.rubocop.yml:2]: 👍

Comment (SBlazhko) [reply guides/code-style/.rubocop.yml:2]: Agree? It configuration now at line 9.

File: guides/code-style/.rubocop.yml (hunk 2 above)
Comment (SBlazhko) [reply guides/code-style/.rubocop.yml]: I think 10 is too small. How about 12 lines?

File: guides/code-style/.rubocop.yml (hunk 3 above)
Comment (SBlazhko) [reply guides/code-style/.rubocop.yml:51]: Because I put the excludes in the general "AllCops" for others this cop enabled by default and it keep unnecessary place now. 
Confirmation that cop "On" by default - https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_style.md#stylenumericliterals

File: guides/code-style/.rubocop.yml (hunk 4 above)
Comment (SBlazhko) [reply guides/code-style/.rubocop.yml:68]: because i put excludes in the general "AllCops"

File: guides/code-style/.rubocop.yml (hunk 2 above)
Comment (IlyaUmanets) [reply guides/code-style/.rubocop.yml]: 👍

**Review by vitaliiprodan — APPROVED** [review]

File: guides/code-style/.rubocop.yml [hunk 5]
```diff
@@ -65,6 +57,13 @@ Style/PercentLiteralDelimiters:
...
-    - 'bin/*'
-    - 'db/schema.rb'
+  EnforcedStyle: single_quotes
+
+Lint/Debugger:
```
Comment (tom-mullen) [inline guides/code-style/.rubocop.yml]: Can we have these in alphabetical order?

File: guides/code-style/.rubocop.yml [hunk 6]
```diff
@@ -19,24 +25,14 @@ Style/BlockComments:
...
-  Exclude:
-    - 'bin/*'
-    - 'db/schema.rb'
-
 Style/FirstParameterIndentation:
   EnforcedStyle: consistent
 
 Style/GuardClause:
-  Enabled: false
+  Enabled: true
```
Comment (tom-mullen) [inline guides/code-style/.rubocop.yml:41]: We use guard clauses extensively. Unless I'm mistaken, this would prevent us from using them. can you confirm?

**Review by tom-mullen — CHANGES_REQUESTED** [review]

File: guides/code-style/.rubocop.yml (hunk 5 above)
Comment (SBlazhko) [reply guides/code-style/.rubocop.yml]: yes, sure i fix it:)

File: guides/code-style/.rubocop.yml (hunk 6 above)
Comment (SBlazhko) [reply guides/code-style/.rubocop.yml:41]: We have been disabled the mention of where you can fix the conditional statement on guard clauses. 
If i set enable: true rubocop will begin to make comments on the places where you can use guard clauses instead if end operator. Also i can remove this statement from file because this cop true by default. 
documentation - https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_style.md#styleguardclause

How you, think what i should do with this?

Comment (tom-mullen) [reply guides/code-style/.rubocop.yml:41]: Makes sense

**Review by tom-mullen — APPROVED** [review]

**Review by IlyaUmanets — APPROVED** [review]
